HealthcareSystem/Backend/Controller/PlacementInARoomAndRenovationPeriod/RenovationPeriodController.cs

```diff
@@ -6,33 +6,34 @@
 
 namespace Controller.PlacementInARoomAndRenovationPeriod
 {
-    public class RenovationPeriodController
-    {
-        private RenovationPeriodService renovationPeriodService = new RenovationPeriodService();
-
-        public Model.Manager.RenovationPeriod ScheduleRenovation(Model.Manager.RenovationPeriod renovationPeriod)
+     public class RenovationPeriodController
         {
-            return renovationPeriodService.ScheduleRenovation(renovationPeriod);
-        }
+            private RenovationPeriodService _renovationPeriodService;
 
-        public bool CancelRenovation(int roomNumber)
-        {
-            return renovationPeriodService.CancelRenovation(roomNumber);
-        }
+            public void AddRenovationPeriod(Model.Manager.RenovationPeriod renovationPeriod)
+            {
+                _renovationPeriodService.AddRenovationPeriod(renovationPeriod);
+            }
 
-        public List<RenovationPeriod> ViewRenovations()
-        {
-            return renovationPeriodService.ViewRenovations();
-        }
+            public void DeleteRenovationPeriod(int roomNumber)
```

Comment:
Why rename Cancel to Delete here? It think cancel was a more logical name for this in the Controller and Service, calling Delete on the Repository is just the way it's currently implemented? (I'm aware it doesn't matter right now anyway since this part of the system will eventually get overhauled by GraphicalEditor anyway, but it just seems like an unnecessary change. Similarly, I'm actually not sure why it was necessary for you to add RenovationPeriod to the database right now anyway? Do you actually need it for anything right now?)

Reply:
RenovationPeriod  is connected with Room so I had to refactor it. I renamed it to be Delete in all layers to be consistent, but if you think that it is better to leave Cancel in Controller and Service I will do so.